docs/desktop-wpf/migration/convert-project-from-net-framework.md

```diff
@@ -50,49 +50,48 @@ The primary challenge of migrating a .NET Framework app to .NET Core is that its
 
 ### Upgrade to `<PackageReference>` NuGet references
 
-Older .NET Framework projects typically list their NuGet dependencies in a *packages.config* file. The new SDK-style project file format references NuGet packages as [`<PackageReference>`](/nuget/consume-packages/package-references-in-project-files) elements in the csproj file itself rather than in a separate config file.
+Older .NET Framework projects typically list their NuGet dependencies in a packages.config file. The new SDK-style project file format references NuGet packages as [`<PackageReference>`](/nuget/consume-packages/package-references-in-project-files) elements in the csproj file itself rather than in a separate config file.
```

Comment:
Wondering why removed the italic format from packages.config file ?
The current style guide says to make files and directories italic.

Reply:
@Youssef1313 The editing team doesn't work that much with our team so they are used to other styling options.